### ISSUE TITLE
Change text colour

### ISSUE THREAD

**Issue by Norric1Admin** [ONSdigital/sdg-indicators#3053]
We need to edit the reporting status text, to turn give one of the categories white letters.
I have started a [feature branch](https://sdg.mango-solutions.com/colour-test/reporting-status/), and tried editing the [CSS](https://github.com/ONSdigital/sdg-indicators/blob/colour_test/assets/css/default.scss) as follows:
- https://github.com/ONSdigital/sdg-indicators/commits/colour_test
- https://github.com/ONSdigital/sdg-indicators/commit/c08bfec2a745785df09f663b8b05e1589e0f8c1f
- https://github.com/ONSdigital/sdg-indicators/commit/1a8b1a41c8a224094e744b63a6e83500ad875681

Please edit the CSS to change `.danger` (line 615) and `.notstarted` (line 625) text colour.

**Comment by Norric1Admin** [comment]
https://github.com/ONSdigital/sdg-indicators/issues/2851

**Comment by SavvasStephanides** [comment]
Hi @Norric1Admin , what are we changing the colors of `.danger` and `.notstarted` to? Thanks.

**Comment by SavvasStephanides** [comment]
I made a small test here: https://github.com/sustainabledevelopment-pseudonesia/sdg-indicators/commit/be3d83ada5a472989ea0eed3e27146076076dd22

I'll soon make a pull request to the ONSDigital site

**Comment by Norric1Admin** [comment]
Testing this at:
https://github.com/ONSdigital/sdg-indicators/edit/colour-test-1/assets/css/default.scss and trying to update the Goal page text (notstarted) to white.

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility tickets](https://trello.com/c/E40VA79G/549-accessibility-tickets)

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility: Sprint 12](https://trello.com/c/HZJO8O4J/576-accessibility-sprint-12)

**Comment by Norric1Admin** [comment]
https://sdg.mango-solutions.com/colour-test-1/3/ :(
https://github.com/ONSdigital/sdg-indicators/commit/e106d1397a86aa436f75cc7f04ad8e2a0f14f907

**Comment by LucyGwilliamAdmin** [comment]
## Description

## Screenshots
**Before**

**After**


## Successfully tested on...
- [ ] Windows: IE11
- [ ] Windows: Chrome
- [ ] Windows: Firefox
- [ ] Windows: Edge
- [ ] macOS: Chrome
- [ ] macOS: Firefox
- [ ] iOS: Safari 9.3
- [ ] iOS: Chrome
- [ ] Samsung: Chrome
- [ ] Samsung: Samsung Internet

## User feedback and problems encountered...

**Comment by Norric1Admin** [comment]
@dougmet Here are the details - we selected `colour-test-1` and successfully updated reporting status. However, the Goal page hasn't updated the text colour and so this is the final step.
- https://github.com/ONSdigital/sdg-indicators/commit/c1549d58c7a6d784fe19e57410355a5e6caf27ec
- https://github.com/ONSdigital/sdg-indicators/commit/e106d1397a86aa436f75cc7f04ad8e2a0f14f907
- https://github.com/ONSdigital/sdg-indicators/commit/b2e6696c259a09231fe3f1dc2a973bb27760e563

**Description**
Change the text of one of the colours to white, so that black contrast is no longer a factor. Use white text for `notstarted` so that it only needs to pass against white. The required contrast values would then be...
___
https://github.com/ONSdigital/sdg-indicators/pull/3068
complete: #1499FC
notstarted: #000D80
notapplicable: #959595

This is one of three colour palettes - the only one which passed at AAA standard.

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Update colours and text in Reporting Status page](https://trello.com/c/K0W8dazi/409-update-colours-and-text-in-reporting-status-page)

**Comment by dougmet** [comment]
If at all possible it'd be better to change the text colour in `custom.css` unless we're making the change for everyone at open-sdg.  Having trouble building the branch because the data branch is no longer on the test server.

I ca no longer trigger rebuilds but if you can give this one a kick then it'll redeploy: https://travis-ci.org/ONSdigital/sdg-data/builds/544221452

**Comment by Norric1Admin** [comment]
@dougmet Should work from https://sdg.mango-solutions.com/colour-test now.

**Comment by LucyGwilliamAdmin** [comment]
@Norric1Admin I think this should be remade from develop as it is since I have removed a lot of the files we no longer need and we're not going to know if something in open-sdg is going to affect the way we do this.

**Comment by Norric1Admin** [comment]
@LucyGwilliamAdmin Agreed, think it is safer to do it that way. Please proceed.

**Comment by LucyGwilliamAdmin** [comment]
@dougmet so [colour-test](https://github.com/ONSdigital/sdg-indicators/tree/colour-test) is up-to-date with develop.

If you look at the [reporting page of the feature branch](https://sdg.mango-solutions.com/colour-test/reporting-status/), the three colours for Reported online, Exploring data sources and Not applicable are right but for some reason Statistics in progress is appearing and I'm not sure where it's come from. We were removing it by changing the reportingstatus.js page but now that the reporting status comes from the data service we no longer use that file. Is there another way to remove Statistics in progress?

Then once that's been fixed we need fix the Reported online style on the goal pages so that the text is white - currently looks like [this](https://sdg.mango-solutions.com/colour-test/1/)

**Comment by Norric1Admin** [comment]
@LucyGwilliamAdmin I removed `inprogress` with https://github.com/ONSdigital/sdg-data/pull/1339
@dougmet The only step left is to fix the Goal page text, but for now please work with @SavvasStephanides on the two remaining Issues: https://trello.com/c/2RhRrhKB/608-make-table-filters-keyboard-compatible-in-ie / https://trello.com/c/2e66QTlh/610-table-label-to-state-which-sort-function-is-applied.

**Comment by AnnCorp** [comment]
Closing various improvements made through Open SDG and page colours etc now configurable.
